openhtf/io/http_api.py

```diff
@@ -119,14 +121,25 @@ def do_GET(self):  # pylint: disable=invalid-name
       """
       result = {'test': util.ConvertToBaseTypes(self.executor.GetState()),
                 'framework': util.ConvertToBaseTypes(self.executor)}
-      self.wfile.write(json.dumps(result))
+      with open(os.devnull, 'w') as f:
+        with contextlib.redirect_stderr(f):  # Don't spam the terminal.
```

Comment:
Ah, it looks like `send_response()` calls `log_request()` which calls `log_message()`. It's recommended in the docs to override `log_message(format, *args)` with something to handle 'custom formatting'. I think we can override `log_message()` or `log_request()` to properly control the terminal output